package.json

```diff
@@ -39,39 +41,6 @@
 				"error",
 				"always"
 			],
-			"object-curly-spacing": [
-				"error",
-				"always"
-			],
-			"multiline-comment-style": "off",
-			"computed-property-spacing": "off",
-			"space-in-parens": "off",
-			"capitalized-comments": "off",
-			"no-lonely-if": "off"
-		},
-		"env": [
-			"es6",
-			"browser",
-			"node",
-			"jquery"
-		]
-	},
-	"eslintConfig": {
-		"extends": "xo",
```

Comment:
This is required

Reply:
I've actually changed this now to switch to eslint from the xo linter due various reasons, mostly support issues.

Reply:
Does not matter that has nothing to do with xojs...it's required for eslint to work correctly